File: SMA.BackEnd/SMA.BackendOps/SMA.BackendOps/Models/Like.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SMA.BackendOps.Models
{
    [Table("Likes", Schema = "SMA")]
    public class Like
    {
        [Key]
        public int like_id { get; set; }
        //FK to User table
        [ForeignKey("User")]
        public int user_id { get; set; }

        [ForeignKey("Post")]
        public int post_id { get; set; }
    }
}

File: SMA.FrontEnd/social-media-app/src/app/classes/like.ts
export class Like {
    like_id: number;
    user_id: number;
    post_id: number;

    constructor(like_id: number, user_id: number, post_id: number) {
        this.like_id = like_id;
        this.user_id = user_id;
        this.post_id = post_id;
    }
}
